public/app/plugins/datasource/prometheus/components/PromQueryField.tsx

```diff
@@ -12,7 +12,7 @@ import {
   BracesPlugin,
 } from '@grafana/ui';
 
-import Prism from 'prismjs';
+import { LanguageMap, Grammar, languages as prismLanguages } from 'prismjs';
```

Comment:
Nit: Don't need `Grammar` with the change mentioned later.

```suggestion
import { LanguageMap, languages as prismLanguages } from 'prismjs';
```